### DISCUSSION TITLE
Any options to use prefixes

### DISCUSSION THREAD

**Discussion by AndreyYolkin** [unocss/unocss#1477] [Q&A] [answered]
Atomic CSS is a good addition to my project, built with vuetify. However, it comes with custom truncated set of atomic classes (flex, margin/padding), which is not compatible with usual AtomicCSS frameworks (like Tailwind).

I am using Tailwind with prefixes, and seems like other frameworks has support for custom prefixes, too.
https://tailwindcss.com/docs/configuration#prefix

Is there any options to add prefix to UnoCSS classes?

**Comment by zyyv** [comment, marked as answer]
we have `un-` prefix, and you can custom it.
https://github.com/unocss/unocss/blob/0f7efcba592e71d81fbb295332b27e6894a0b4fa/packages/preset-mini/src/index.ts#L49-L54

**Reply by AndreyYolkin** [reply]
Great, thanks!

**Reply by bonbisu** [reply]
I tried to pass this on `unocss.config.ts` but still not matching `tw-`

```ts
// ...

export default defineConfig({
  presets: [
    presetUno({
      variablePrefix: 'tw-'
     }),
    // ... other plugins
  ]
})
```

Nothing changes, only matching tailwind classes without prefix.

I also tried to use the default prefix 'un-' but do not work too.

> Additional info: I'm using Vitesse, maybe I have to config on `vite.config.ts`, but i don't know how.

**Reply by zyyv** [reply]
Sorry, should `prefix` option. 😅  Corrected.  [Playground](https://uno.antfu.me/play/?html=DwEwlgbgBAxgNgQwM5ILwCIAuB3AtJgUwA9NcAnAkdAPgCgooB7AO1xAIFsFmQoB2qGGZJM3TPygAzAK5w4AT1jSRjDmABeCAEZwCtYAHpwEakA&config=JYWwDg9gTgLgBAbzgEwKYDNgDtUGEJaYDmANHGFKgM6owCCMMUwARgK4zDoCeZF1tAJIBjAlT6UaMAKpYIcAL5x0UCCDgByNnOFUqGgFAHUAD0iwUGAIZsANvDSYc%2BQsCIAKBAbhwod6gBccADawRrCbFQwagC0fraoGmRIorbQQRqUyBqKALq5JN5wVAAW0DARMFRBXj4%2B4ZHRIDGl5ZUaGTCmMDG2RHBdJj3QVlhEqHBlAG6oUAGDPV1WtklFCoU%2B-FLVIUWbkrSyEJ7klJgmnQDuMTkKAJQbdVu0DEysHFzc7g97pwIwIjEnl%2BPiowmWqCCAEYAHQAJkedTgwmQWAyJUYYGqAHpsdQQDDStjVkj7o8CgZ7gYgA&options=N4IgTgpgzgDg9gOygSwG4RALgGYEMA2UEAvkA)
